src/main/java/by/foo/inspector/SongInspector.java

```diff
@@ -0,0 +1,85 @@
+package by.foo.inspector;
+
+import com.google.common.collect.ImmutableSet;
+import ealvatag.audio.AudioFile;
+import ealvatag.audio.AudioFileIO;
+import ealvatag.audio.exceptions.CannotReadException;
+import ealvatag.audio.exceptions.InvalidAudioFrameException;
+import ealvatag.tag.FieldKey;
+import ealvatag.tag.NullTag;
+import ealvatag.tag.Tag;
+import ealvatag.tag.TagException;
+import org.apache.commons.io.FilenameUtils;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.file.FileSystems;
+import java.nio.file.Path;
+
+
+public class SongInspector extends MediaFileInspector {
+
+  public SongInspector() {
+    try {
+      inspect(getCurrentDirectory());
+    } catch (IOException | CannotReadException | InvalidAudioFrameException | TagException e) {
+      e.printStackTrace();
+    }
+  }
+
+  @Override
+  File getCurrentDirectory() {
+    Path path = FileSystems.getDefault().getPath("").toAbsolutePath();
```

Comment:
`FileSystems.getDefault().getPath("").toAbsolutePath().toFile()`?

Not sure if `.toAbsolutePath()` is even needed.

Reply:
an NPE arises wihout it

Reply:
NPE is better than wrong behavior. What would you do with `new File("null")`? Check for null explicitly or something...